streaming/sender.c

```diff
@@ -203,6 +203,18 @@ void rrdpush_clean_encoded(stream_encoded_t *se)
         freez(se->kernel_version);
 }
 
+static inline long int parse_stream_version_for_errors(char *http)
+{
```

Comment:
A suggestion, instead to use `strlen` for each one of the next `memcp` calls, I suggest to use `sizeof`, because it will avoid calls and it will add the  `NULL` at the end.